jenkins-dsl/cassandra_job_dsl_seed.groovy

```diff
@@ -54,7 +54,7 @@ def dtestTargets = ['dtest', 'dtest-novnode', 'dtest-offheap', 'dtest-large']
 if(binding.hasVariable("CASSANDRA_DTEST_TEST_TARGETS")) {
     dtestTargets = "${CASSANDRA_DTEST_TEST_TARGETS}".split(",")
 }
-def dtestDockerImage = 'spod/cassandra-testing-ubuntu18-java11'
+def dtestDockerImage = 'nastra/cassandra-testing-ubuntu1910-java11'
```

Comment:
I need to double-check it's ok a non-committer dockerhub image is used in the building. It's not a release so I suspect it is ok. 

@spodkowinski are you aware of what's permitted here?

(later on it might make sense if we use the apache dockerhub, i don't think names like  `https://hub.docker.com/r/apache/cassandra-testing-ubuntu1910-java11` will confuse end users)

Reply:
I would like to come back to this, and find a way we can keep images (even those just for testing) under `https://hub.docker.com/r/apache/`